lib/utils/format.ts

```diff
@@ -116,9 +116,8 @@ export const formatPaymentType = (paymentMethod: PaymentType): string => {
     MASTERCARD: 'MasterCard (Office)',
     VISA: 'Visa (Office)',
     CASH: 'Cash',
-    DEBIT: 'Debit',
+    DEBIT: 'Interac - Debit (Office)',
```

Comment:
This line looks good.

Sorry, this should've been addressed in an earlier PR but this function isn't really a formatting function - it's used to map a payment enum to a string representing the payment type. Furthermore, this util function is only used in one place, which is the accounting report generation (whereas the other true formatting functions are used in many places throughout the app). This map should just live inside the `generateAccountantReport` resolver.